### PR TITLE
Add Supports to KWin 5.26

### PR DESCRIPTION
**Pull request by matinlotfali**

### CODE REVIEW

**Comment by matinlotfali** [comment]
Currently, there is a difference between header files of KWinEffect API between KDE Neon **Stable** and **Unstable**. I don't know how to write the effect to work on both of them.

I reported this on KDE.org: https://bugs.kde.org/show_bug.cgi?id=456155

File: shapecorners.cpp [hunk 1]
```diff
@@ -71,15 +71,20 @@ ShapeCornersEffect::ShapeCornersEffect() : KWin::Effect(), m_shader(nullptr)
     if (file.open(QFile::ReadOnly))
     {
         QByteArray frag = file.readAll();
-        m_shader = KWin::ShaderManager::instance()->generateCustomShader(KWin::ShaderTrait::MapTexture, QByteArray(), frag);
+        auto shader = KWin::ShaderManager::instance()->generateCustomShader(KWin::ShaderTrait::MapTexture, QByteArray(), frag);
+#if KWIN_EFFECT_API_VERSION >= 235
```
Comment (alex1701c) [inline shapecorners.cpp:75]: You could better use QT_VERSION_CHECK, that does the bitshifting for you

**Review by alex1701c — APPROVED** [review]
Looks good from my side.

File: shapecorners.cpp (hunk 1 above)
Comment (matinlotfali) [reply shapecorners.cpp:75]: Can you elaborate?

Comment (alex1701c) [reply shapecorners.cpp:75]: Oh nevermind, KWin does the versioning really weird :/

**Comment by matinlotfali** [comment]
The MR is merged. So I will also merge this PR.

**Pull request merged**